index.html

```diff
@@ -20,7 +20,40 @@
           <h1>Hans Bernadin</h1>
           <a href="#"><i class="fas fa-bars"></i></a>
         </nav>
+        <section class="showcase">
+          <h2>
+            I'm Hans <br />
+            Glad to see you!
+          </h2>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <h3>Let's Connect</h3>
```

Comment:
- [ ] Kindly add the `Let's Connect`  in a `<a>` tag. It will help the user to navigate.